srcpkgs/nagios/template

```diff
@@ -0,0 +1,64 @@
+# Template file for 'nagios'
+pkgname=nagios
+version=4.4.6
+revision=1
+build_style=gnu-configure
+
+_nagios_user="_nagios"
+_nagios_group="_nagios"
+_instdir="/usr/share/nagios"
+_bindir="/usr/bin"
+_vardir="/var/nagios"
+_confdir="/etc/nagios"
+_httpdconfdir="/etc/webapps/$pkgname/"
+_checkresultdir="/var/nagios/spool/checkresults"
+_cgibindir="/usr/libexec/nagios/cgi"
+
+configure_args="
+	--with-nagios-user=$_nagios_user
+	--with-nagios-group=$_nagios_group
+	--prefix=$_instdir
```

Comment:
Doesn't those flags set by `gnu-configure`?

Reply:
Which flags specifically? I can test omitting `--prefix` and seeing what happens.

Reply:
I tested with `--prefix` removed from `configure_args` and it appears to work as expected!

Reply:
Those flags are set by `gnu-configure`:

https://github.com/void-linux/void-packages/blob/ab8ff2e758d8d8d9314c296596ad86dca4e417a7/common/environment/configure/gnu-configure-args.sh#L7-L17
https://github.com/void-linux/void-packages/blob/ab8ff2e758d8d8d9314c296596ad86dca4e417a7/common/environment/configure/gnu-configure-args.sh#L37